.ci.yaml

```diff
@@ -1567,6 +1567,16 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Linux_android hello_world_impeller
```

Comment:
Whoops. The `Linux_android` bit will make this run on a Moto G4.

Changing the `Linux_android` to `Linux_pixel_7pro` will make this run on a Pixel 7 Pro.